app/controllers/user_blocks_controller.rb

```diff
@@ -0,0 +1,78 @@
+class UserBlocksController < ApplicationController
+  before_action :check_sign_in_status
+  after_action :verify_authorized
+
+  def show
+    skip_authorization
+
+    if current_user.blocking?(params[:blocked_id].to_i)
+      render plain: "blocking"
```

Comment:
I was wondering why some endpoints return strings and some JSON, maybe they should be a bit more uniform?